### PR TITLE
Powershell release StandbyPool 2024-03-01

### PR DESCRIPTION
**Pull request by fangru2023**
### ChangeLog
1. Added new Cmdlets: 
```
Get-AzStandbyContainerGroupPoolStatus

Get-AzStandbyVMPoolStatus
```
2. Update existing Cmdlets
```
New-AzStandbyVMPool by add new parameter -MinReadyCapacity to support new added properties in put call.
```
3. Deprecated Cmdlets
```
Get-StandbyVMPoolVM
```

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

## Mandatory Checklist

- Please choose the target release of Azure PowerShell. (⚠️**Target release** is a different concept from **API readiness**. Please click below links for details.)
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] No need for a release

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by dolauli** [comment]
@fangru2023 Please add a file like https://github.com/Azure/azure-powershell/blob/main/tools/StaticAnalysis/Exceptions/Az.Cdn/UXMetadataIssues.csv for your module to suppress the UX failures. Please check the failed CI job for what you should input in the file.

File: src/StandbyPool/StandbyPool.Autorest/README.md [hunk 1]
```diff
@@ -122,28 +144,73 @@ directive:
...
+  - where:
+      verb: Get
+      subject: StandbyContainerGroupPoolStatus
+      parameter-name: Version
+    hide: true
+    set:
+      default:
+        script: "'latest'"
+
+  # Rename Get-StandbyVirtualMachine to StandbyVM
```
Comment (dolauli) [inline src/StandbyPool/StandbyPool.Autorest/README.md:208]: Based on the comments, it should be an operation of renaming. But actually, it is to remove the cmdlet. Is it expected? Besides this cmdlet is exported in the last version.

**Review by dolauli — CHANGES_REQUESTED** [review]

File: src/StandbyPool/StandbyPool.Autorest/README.md (hunk 1 above)
Comment (fangru2023) [reply src/StandbyPool/StandbyPool.Autorest/README.md:208]: Hi @dolauli, it's expected, as this cmdlets is not functioning for right now.

**Comment by fangru2023** [comment]
cc: @mimckitt @jiaosummer

File: src/StandbyPool/StandbyPool.Autorest/README.md [hunk 2]
```diff
@@ -40,7 +41,7 @@ try-require:
   - $(repo)/specification/standbypool/resource-manager/readme.powershell.md
 
 # For new RP, the version is 0.1.0
-module-version: 0.1.0
+module-version: 0.2.0
```
[the remark this answers is not in the document]
Comment (fangru2023) [reply src/StandbyPool/StandbyPool.Autorest/README.md]: Thanks for pointing out, addressed.

**Comment by fangru2023** [comment]
> @fangru2023 Please add a file like https://github.com/Azure/azure-powershell/blob/main/tools/StaticAnalysis/Exceptions/Az.Cdn/UXMetadataIssues.csv for your module to suppress the UX failures. Please check the failed CI job for what you should input in the file.

Hi @msJinLei , and @dolauli ,

Thanks for reviewing, address this comment and updated pr.

**Review by msJinLei — CHANGES_REQUESTED** [review]
Please provide a change log for your PR

**Comment by fangru2023** [comment]
> Please provide a change log for your PR

Thanks for reviewing, updated PR description with ChangeLog.

**Review by msJinLei — APPROVED** [review]

File: src/StandbyPool/StandbyPool.Autorest/README.md (hunk 1 above)
Comment (msJinLei) [reply src/StandbyPool/StandbyPool.Autorest/README.md:208]: > Hi @dolauli, it's expected, as this cmdlets is not functioning for right now.

@fangru2023 The issue is the comment should be consistant with the actual directives

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by fangru2023** [comment]
Thanks @msJinLei, updated pr to address your comment.

File: src/StandbyPool/StandbyPool.Autorest/README.md (hunk 1 above)
Comment (fangru2023) [reply src/StandbyPool/StandbyPool.Autorest/README.md]: Addressed comments. Thanks for suggestion.

**Review by dolauli — APPROVED** [review]

**Review by msJinLei — APPROVED** [review]

**Comment by fangru2023** [comment]
Hi @dolauli, and @msJinLei, do you know when will our powershell package release, do we need start a release?

**Pull request merged**